2_Foundations/Serialization.md

```diff
@@ -84,11 +84,11 @@ void writeValue(const char* propertyName, T value);
 
 ## vsg::ObjectFactory
 
-When writing out objects you can simply call **object->write(output)** and the appropriate serialization will be invoked, but when you need to serialize a file back in, the appropriate objects have to be created before their **object->read(input)** method can be invoked to read the object members.  The way the VulkanSceneGraph provides a means for creating objects on demand is via the [vsg::ObjectFactory](https://github.com/vsg-dev/VulkanSceneGraph/tree/master/include/vsg/io/ObjectFactory.h#L24) singleton class, where only one instance of the Factory exists.  The vsg::ObjectFactory is an example of the [Factory Method Design Pattern](https://en.wikipedia.org/wiki/Factory_method_pattern) and [Singleton Design Pattern](https://en.wikipedia.org/wiki/Singleton_pattern).
+When writing out objects you can simply call **object->write(output)** and the appropriate serialization will be invoked, but when you need to serialize back into a file, the appropriate objects have to be created before their **object->read(input)** method can be invoked to read the object members. The way the VulkanSceneGraph provides a means for creating objects on demand is via the [vsg::ObjectFactory](https://github.com/vsg-dev/VulkanSceneGraph/tree/master/include/vsg/io/ObjectFactory.h#L24) singleton class, where only one instance of the Factory exists. The vsg::ObjectFactory is an example of the [Factory Method Design Pattern](https://en.wikipedia.org/wiki/Factory_method_pattern) and [Singleton Design Pattern](https://en.wikipedia.org/wiki/Singleton_pattern).
```

Comment:
The original is correct.  I think you must have scanned it, but not actually read what the paragraph was trying to convey.

Reply:
Yes you're right.